### DISCUSSION TITLE
Expand Snapcraft app metadata?

### DISCUSSION THREAD

**Discussion by dstotijn** [goreleaser/goreleaser#2952] [Ideas, Issue Triage, and General discussions]
Currently, Snapcraft app metadata allows setting the following fields: https://github.com/goreleaser/goreleaser/blob/0bb1c99074f241731a63b8da542bdfb79eb2e01b/internal/pipe/snapcraft/snapcraft.go#L57-L60

There are more options available though ([ref](https://snapcraft.io/docs/snapcraft-app-and-service-metadata)), for instance `install-mode`, which I would love to be able to use to disable automatic daemon startup control.

@caarlos0 You reckon it would be worth it for me to create a PR to add the missing keys from [the Snapcraft docs](https://snapcraft.io/docs/snapcraft-app-and-service-metadata)? Or was it an intentional decision to only support a subset of the available app and service metadata?

**Comment by caarlos0** [comment]
hey @dstotijn , the options available today likely were the ones available when it was first implemented, so I think we can add those new ones :)

prs very welcome 🤘

**Reply by dstotijn** [reply]
PR: https://github.com/goreleaser/goreleaser/pull/2955
